### PR TITLE
add changelog for release-1.16

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add changelog for release-1.16

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.16.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Refer to the link for more details.([#5308](https://github.com/kubeedge/kubeedge/pull/5308), [#5326](https://github.com/kubeedge/kubeedge/pull/5326))
+
+### Integrate Redis and TDengine Database in DMI Data Plane
+
+Integrate redis and tdengine database in DMI data plane. The mapper project generated by mapper-framework has build-in ability to push data to redis and tdengine database. Users can push data directly through configuring device instance files.
+
+Refer to the link for more details.([#5064](https://github.com/kubeedge/kubeedge/pull/5064))
+
+### New USB Camerica Mapper
```
Comment (fisherxu) [inline CHANGELOG/CHANGELOG-1.16.md:75]: Camera? @luomengY

**Comment by zhiyingfang2022** [comment]
Should important bug fixes also be added, or can it be divided into categories according to new features, optimization, and bug fixes?

File: CHANGELOG/CHANGELOG-1.16.md (hunk 1 above)
Comment (luomengY) [reply CHANGELOG/CHANGELOG-1.16.md:75]: > Camera

Yes, this should be Camera.

Comment (Shelley-BaoYue) [reply CHANGELOG/CHANGELOG-1.16.md]: fixed.

**Comment by fisherxu** [comment]
> Should important bug fixes also be added, or can it be divided into categories according to new features, optimization, and bug fixes?

Sure, we can add them later.

File: CHANGELOG/CHANGELOG-1.16.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    In KubeEdge v1.16, the node upgrade API was implemented. Users can remotely upgrade edge nodes in batches. The cloud-edge task architecture handles upgrade task flow and supports unified timeout processing, concurrency control, and subtask management, among other capabilities.
+
+- **KubeEdge version compatibility testing**
+
+    KubeEdge v1.16 provides KubeEdge version compatibility testing, which avoids problems caused by incompatible cloud-edge versions during the upgrading process.
+
+Refer to the link for more details. ([#5330](https://github.com/kubeedge/kubeedge/pull/5330), [#5229](https://github.com/kubeedge/kubeedge/pull/5229), [#5289](https://github.com/kubeedge/kubeedge/pull/5289))
+
+### Support Images PrePull on Edge Nodes
```
Comment (fisherxu) [inline CHANGELOG/CHANGELOG-1.16.md:33]:
```suggestion
### Alpha Implementation of Images PrePull on Edge Nodes 
```

Comment (Shelley-BaoYue) [reply CHANGELOG/CHANGELOG-1.16.md]: done.

**Comment by Shelley-BaoYue** [comment]
cc @fisherxu @WillardHu @zhiyingfang2022

**Pull request merged**